modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/viewmodel/DiscoverViewModel.kt

```diff
@@ -71,6 +71,7 @@ class DiscoverViewModel @Inject constructor(
             categoryFeed = null,
             isLoading = true,
             isError = false,
+            openStaffPicks = false,
```

Comment:
Navigation is a one-off event that is a side effect and mustn't be part of the state. Having it here can lead to many ambiguities when there are many such fields and more then 1 get set to `true`. Additionally, this makes state hard to manage as it requires setting values to `false` from the UI after navigation occurs. Please use a different solution to the navigation.